### PR TITLE
Moving UDP Module to an External Repository 

### PR DESCRIPTION
**Pull request by MohamedSabthar**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+plugins {
+    id "com.gradle.enterprise" version "3.2"
+}
+
+rootProject.name = 'udp'
+include(':build-config:checkstyle')
+include ':udp-native', ':udp-test-utils', ':udp-ballerina'
+include 'udp-test-utils'
+include 'udp-native'
```
Comment (BuddhiWathsala) [inline settings.gradle:18]: We don't need these lines.

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

**Comment by ThishaniLucas** [comment]
Please follow https://chris.beams.io/posts/git-commit/ for the commit messages.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/module-ballerina-udp'
```
Comment (ThishaniLucas) [inline build.gradle:62]: This is not necessary. Referring to the same repo.

File: udp-ballerina/Ballerina.toml [hunk 3]
```diff
@@ -1,27 +1,27 @@
 [package]
 org = "ballerina"
 name = "udp"
-version = "@toml.version@"
+version = "0.8.0"
```
Comment (Maninda) [inline udp-ballerina/Ballerina.toml:4]: Seems this is mistakenly committed. Refer the other such changes in `Ballerina.toml` file as well.

File: README.md [hunk 4]
```diff
@@ -1 +1,62 @@
...
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/udp.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsocket)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The UDP library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This module provides an implementation for sending/receiving messages to/from another application process (local or remote) for connectionless protocols. 
+
+For more information on the operations supported go to [The UDP Module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/udp/).
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/udp-socket-client.html).
```
Comment (Maninda) [inline README.md:15]: Please make sure we add the the BBE (Ballerina by Example) later with this name.

File: settings.gradle (hunk 1 above)
Comment (MohamedSabthar) [reply settings.gradle:18]: will remove

File: build.gradle (hunk 2 above)
Comment (MohamedSabthar) [reply build.gradle]: this entry is removed now

File: README.md (hunk 4 above)
Comment (MohamedSabthar) [reply README.md]: BBE removed from README

File: udp-ballerina/Ballerina.toml (hunk 3 above)
Comment (MohamedSabthar) [reply udp-ballerina/Ballerina.toml]: File reverted

**Review by Maninda — APPROVED** [review]

**Pull request merged**